README.md

```diff
@@ -1,33 +1,45 @@
-Jovian NixOS
-============
+NixOS on the Steam Deck
+=======================
 
-> **Jovian**  
-> “Relating to [...] Jupiter or the class of [...] which Jupiter belongs.”
+A set of packages and configurations for running NixOS on the [Steam Deck](https://www.steamdeck.com).
+This repo is also useful for obtaining a Deck-like experience on other X86-64 devices.
 
-*By jove! What a weird name!*
+Installation
+------------
 
-This is a "jupiter-compatible" set of packages and configurations for NixOS.
+You can build an installation ISO for the Steam Deck with `nix-build -A <flavor>`, where `<flavor>` is one of the following:
 
-> What's Jupiter?
+- `isoMinimal`: Minimal non-graphical installation environment
+- `isoGnome`: Graphical installation ISO with GNOME
+- `isoPlasma`: Graphical installation ISO with KDE
 
-[There's a disambiguation page that won't help you](https://en.wikipedia.org/wiki/Jupiter_(disambiguation)).
+To enter the boot menu, power off the Steam Deck then hold down `Volume-` and tap the `Power` button.
 
-> But really?
+Follow the [Installing NixOS](https://nixos.org/manual/nixos/unstable/index.html#sec-installation) section of the NixOS Manual.
+When configuring the system, import `./modules` from this repo in your configuration, then enable the Steam Deck-specific hardware configurations with `jovian.devices.steamdeck.enable = true;`.
```

Comment:
```suggestion
Follow the [Installing NixOS](https://nixos.org/manual/nixos/unstable/index.html#sec-installation) section of the NixOS Manual.
When configuring the system, import `./modules` from this repo in your configuration.

When used on a Steam Deck, enable the Steam Deck-specific hardware configurations with `jovian.devices.steamdeck.enable = true;`.
```

Reply:
Since our ISO shouldn't be used on other hardware, let's try splitting the section to make it less hairy.